src/vision/VisionProcessor.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef VISION_PROCESSOR_H
+#define VISION_PROCESSOR_H
+#include <ros/ros.h>
```

Comment:
You include this in Vision.cpp using quotes yet here you use the <>. Is it in the std library path or not?